components/eam/src/chemistry/aerosol/wetdep.F90

```diff
@@ -1149,267 +1149,146 @@ end subroutine wetdepa_v2
 
 
 !==============================================================================
+!     flux_precnum_vs_flux_prec_mp = precipitation number flux at the cloud base [drops/m^2/s]
+!     Options of assuming log-normal or marshall-palmer raindrop size distribution
+!
       function flux_precnum_vs_flux_prec_mpln( flux_prec, jstrcnv )
       real(r8) :: flux_precnum_vs_flux_prec_mpln
```

Comment:
Do we know the units of these fluxes?

Reply:
pushed a new commit with units added and single-letter variables re-defined.